test_conformance/basic/test_progvar.cpp

```diff
@@ -1518,6 +1541,18 @@ static int l_user_type( cl_device_id device, cl_context context, cl_command_queu
 // Test support for variables at program scope. Miscellaneous
 int test_progvar_prog_scope_misc(cl_device_id device, cl_context context, cl_command_queue queue, int num_elements)
 {
+    cl_bool skip{ CL_FALSE };
+    auto error = should_skip(device, skip);
+    if (CL_SUCCESS != error)
+    {
+        return TEST_FAIL;
+    }
+    if (skip)
+    {
+        log_info("Skipping progvar_prog_scope_misc since it is optionally not "
+                 "supported on this device");
```

Comment:
Please add `\n` in the end of `log_info` message

Reply:
Fixed